<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MulDMJForm</class>
 <widget class="QWidget" name="MulDMJForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1,1" columnminimumwidth="0,0,0">
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView_60PR"/>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="graphicsView_HM100PR"/>
   </item>
   <item row="0" column="2">
    <widget class="QGraphicsView" name="graphicsView_100PR"/>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graphicsView_DT"/>
   </item>
   <item row="1" column="1">
    <widget class="QGraphicsView" name="graphicsView_100PRW1"/>
   </item>
   <item row="1" column="2">
    <widget class="QGraphicsView" name="graphicsView_100PRW2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
